content/api/tracking-domains.apib

```diff
@@ -354,10 +354,15 @@ Delete an existing tracking domain.
 
 
 
-### List All Tracking Domains [GET /v1/tracking-domains]
+### List All Tracking Domains [GET /v1/tracking-domains{?default,subaccounts,delimiter}]
 
 Returns an array with all tracking domains. Each item in the array is a full tracking domain object.
 
++ Parameters
+    + default = `false` (optional, boolean) ... Restrict results to default domains when true.
+    + subaccounts (optional, list) ... Delimited list of subaccounts to include in results.
```

Comment:
just curious why are we handling subaccounts as a query param rather than just using X-MSYS-SUBACCOUNT header?

I get it if we're fetching multiple at a time, but I thought the point of this endpoint was to fetch the correct tracking domain for a particular tenant/cid/sid?

Reply:
This is primarily for handling cases where a subaccount does not have a default tracking domain and needs to fallback to the master. This way we get the fallback domain without doing another query and can cache the domain for the master account separately from the subaccount